src/main/resources/application.yaml

```diff
@@ -146,6 +149,9 @@ proxy:
 
 reconciliation:
   api-key: ${RECONCILIATION_API_KEY}
+  report:
+      mail-from: ${SMTP_USERNAME}
+      mail-recipients: ${REPORTS_RECIPIENTS}
```

Comment:
using same values like other reports

Reply:
i guess we want to keep it separate in case recipients need to be different

Reply:
that is why I created different properties for now.